governance/helpers/logger.js

```diff
@@ -0,0 +1,25 @@
+const Sentry = require('@sentry/node')
+
+// init sentry
+if (process.env.SENTRY_DSN) {
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN,
+  })
+}
+
+const symbols = {
+  error: '❌ ',
+  warning: '⚠️ ',
+}
+
+const log = (msg, level = 'info') => {
+  //log to stdout
+  console.log(`${symbols[level] || ''}[${level}]: ${msg}`)
+
+  // send to sentry
+  if (process.env.SENTRY_DSN) {
```

Comment:
logs will be sent to sentry only if `SENTRY_DSN` is exported to the shell. Should document this maybe but not sure where ?

Reply:
I think that's ok.